03-transactions.md

```diff
@@ -109,7 +111,40 @@ If a revoked commitment transaction is published, the other party can spend this
 
 #### `to_remote` Output
 
-This output sends funds to the other peer and thus is a simple P2WPKH to `remotepubkey`.
+This output sends funds to the other peer, thus is not encumbered by a
+revocation private key.
+
+If `option_simplified_commitment` applies to the commitment transaction, the `to_remote` output is delayed similarly to the `to_local` output, and is to a fixed key:
+
+        `to_self_delay`
+        OP_CSV
+        OP_DROP
+        <remote_pubkey>
+
+The output is spent by a transaction with `nSequence` field set to `to_self_delay` (which can only be valid after that duration has passed) and witness:
+
+    <remote_sig>
+
+Otherwise, this output is a simple P2WPKH to `remotepubkey`.
+
+
+#### `to_local_pushme` and `to_remote_pushme` Output (option_simplified_commitment)
+
+This output can be spent by the local and remote nodes respectivey to provide incentive to mine the transaction, using child-pays-for-parent.  They are only added if the `to_local` and `to_remote` outputs exist, respectively.
+
+    OP_DEPTH
+    OP_IF
+        <pubkey> OP_CHECKSIG
+    OP_ELSE
+        10 OP_CSV
+    OP_ENDIF
+
+The `<pubkey>` is `<local_delayedpubkey>` to `to_local_pushme` and
+`<remote_delayedpubkey>` for `to_remote_pushme`.  The output amount is
+1000 satoshi, to encourage spending of the output.  Once the
```

Comment:
Can you add a rationale for these two ways of spending it?
* Why not only let the output's owner spend it?
* Why not let anybody spend it always, with no timelock?

Reply:
Updated.  Thanks!